collections/corine-land-cover-accounting-layers/README.MD

```diff
@@ -0,0 +1,271 @@
+# CORINE Land Cover Accounting Layers
+
+## Short description
+ The CORINE Land Cover (CLC) Accounting Layers are CLC status layers modified for the purpose of consistent statistical analysis in the land cover change accounting system at EEA. 
+ The modification combines CLC status and change layers in the 100m raster form in order to create homogeneous quality time series of CLC / CLC-change layers for accounting purposes.
+ The CLC inventory consists of 44 land cover and land use classes derived from a series of satellite missions since it was first established. 
+ There are altogether 5 mapping inventories implemented since 1986, producing five status layers (CLC1990, CLC2000, CLC2006, CLC2012, CLC2018), 4 CLC-Change layers for the corresponding periods (1990-2000, 2000-2006, 2006-2012, 2012-2018) and 4 CLC accounting layers for 2000, 2006, 2012 and 2018. 
+
+## Band information
+The `CLC-ACC` band values correspond to a land cover and land use classification scheme with class values ranging from 111 to 999. Please see the nomenclature and coloring scheme in the table below. For more details about the classes, please see the official [CORINE Land Cover nomenclature guidelines](https://land.copernicus.eu/user-corner/technical-library/corine-land-cover-nomenclature-guidelines/html/index.html) or the illustrated [nomenclature guidelines document](https://land.copernicus.eu/user-corner/technical-library/corine-land-cover-nomenclature-guidelines/docs/pdf/CLC2018_Nomenclature_illustrated_guide_20190510.pdf). 
+ 
+A visualisation script can be found in our [custom scripts repository](https://custom-scripts.sentinel-hub.com/copernicus_services/corine_land_cover_accounting_layer/).
+
+### Table 2: CORINE Land Cover nomenclature and colouring scheme.
+
+<table>
+  <thead>
+    <tr>
+      <th>Value</th>
+      <th>Color</th>
+      <th>Label</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>111</td>
+      <td style="background-color: #e6004d;"></td>
```

Comment:
The colours are not showing for me.

Reply:
just here on GitHub or even as an html page  ?

seems here on GitHub  the inline html styling in the README is not rendered , thats why the color styling doesnt show .  
However the colors should  show up on the created html page, like this one,  https://collections.sentinel-hub.com/corine-land-cover-accounting-layers/readme.html ..   

I guess that should be okay for now ? . I will later on try to fix the styling so that the colors show up